packages/office-ui-fabric-react/src/components/DocumentCard/DocumentCardActivity.tsx

```diff
@@ -35,14 +35,16 @@ export class DocumentCardActivity extends React.Component<IDocumentCardActivityP
 
     return (
       <div className={ css('ms-DocumentCardActivity-avatar', styles.avatar) }>
-        { person.initials && (
-          <div className={ css('ms-Persona-initials', styles.avatarInitials) } role='presentation'>
```

Comment:
If we aren't using the avatarInitials, can it be removed from the sass file? Also, does the resulting html use the class ms-Persona-initials? Just want to make sure anyone who is relying on that class won't see this break.